internal/resources/metal/vrf_bgp_dynamic_neighbor/resource_schema.go

```diff
@@ -0,0 +1,58 @@
+package vrfbgpdynamicneighbor
```

Comment:
In the docs/spec I think this is generally referred to as "BGP dynamic neighbor", so maybe we can drop the "VRF" here?